build.gradle

```
@@ -1,135 +1,24 @@
-import java.util.stream.Collectors
-
 plugins {
 	id 'java'
```

java-library
```suggestion
	id 'java-library'
```

https://docs.gradle.org/current/userguide/java_library_plugin.html

---

The Java Library plugin expands the capabilities of the Java Plugin (java) by providing specific knowledge about Java libraries. In particular, a Java library exposes an API to consumers (i.e., other projects using the Java or the Java Library plugin). All the source sets, tasks and configurations exposed by the Java plugin are implicitly available when using this plugin.

---

[java vs java-library](https://discuss.gradle.org/t/when-should-we-use-java-plugin-and-when-java-library-plugin/25377/4)

멀티모듈에서 java library를 사용하는게 이점이 더 많은거 같네요?
이거 한번 로컬에서 돌려보고 커밋 날리겠습니다!